java10/scripts.js

```diff
@@ -0,0 +1,145 @@
+var newGameBtn = document.getElementById('js-newGameButton'),
+    newGameElem = document.getElementById('js-newGameElement'),
+    pickElem = document.getElementById('js-playerPickElement'),
+    resultsElem = document.getElementById('js-resultsTableElement');
+
+newGameBtn.addEventListener('click', newGame);
+
+var gameState = 'notStarted',
+    player = {
+        name: '',
+        score: 0
+    },
+    computer = {
+        score: 0
+    };
+
+function setGameElements() {
+    switch(gameState) {
+        case 'started':
+            newGameElem.style.display = 'none';
+            pickElem.style.display = 'block';
+            resultsElem.style.display = 'block';
+        break;
+        case 'ended':
+            newGameBtn.innerText = 'Try again';
+            newGameElem.style.display = 'block';
+            pickElem.style.display = 'none';
+            resultsElem.style.display = 'block';
+        break;
+        case 'notStarted':
+        default:
+            newGameElem.style.display = 'block';
+            pickElem.style.display = 'none';
+            resultsElem.style.display = 'none';
+    }
+}
+
+setGameElements();
+
+var playerPointsElem = document.getElementById('js-playerPoints'),
+    playerNameElem = document.getElementById('js-playerName'),
+    computerPointsElem = document.getElementById('js-computerPoints');
+
```

Comment:
staraj się żeby kod nie leżał luźno pomiędzy deklaracjami funkcji. ja bym to przeniósł wyżej. `setGameElements()` też.